src/src/tx_main.cpp

```diff
@@ -354,9 +354,12 @@ void ICACHE_RAM_ATTR SendRCdataToRF()
     }
   }
 
+  // artificially inject the nonce on data packets, this will be overwritten with the CRC after it's calculated
+  Radio.TXdataBuffer[0] |= NonceFHSSresult << 2;
+
   ///// Next, Calculate the CRC and put it into the buffer /////
   uint16_t crc = ota_crc.calc(Radio.TXdataBuffer, 7, CRCInitializer);
-  Radio.TXdataBuffer[0] |= (crc >> 6) & 0b11111100;
+  Radio.TXdataBuffer[0] = (Radio.TXdataBuffer[0] & 0b11) | ((crc >> 6) & 0b11111100);
```

Comment:
Ok, so it's just style, but everyone should be able to understand the use of the combo assignment operators and they provide less opportunity for mistakes when they are appropriate. Recommended for code quality and readability.